docs/reference/api/lightning_adapter.txt

```diff
@@ -0,0 +1,96 @@
+##############################
+ determined.pytorch.lightning
+##############################
+
+.. _lightning-adapter:
+
+*********************************************
+ ``determined.pytorch._lightning.PLAdapter``
+*********************************************
+
+Pytorch Lightning Adapter, or PLAdapter for short, provides a quick way
+to train your Pytorch Lightning models with all the Determined features,
+such as mid-epoch preemption, simple distributed training interface,
+simple job submission to the Determined cluster, and so on.
+
+PLAdapter is built on top of our :ref:`PyTorchTrial <pytorch-trial>`
+API, which has a built-in training loop that integrates with the
+Determined features. However, it only supports `LightningModule
+<https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_module.html>`_.
+To migrate your code from the `Trainer
+<https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html>`_,
+please read more about :ref:`PyTorchTrial <pytorch-trial>` and
+:ref:`experiment-configuration`.
+
+PLAdapter takes ``LightningModule`` from your Pytorch Lightning code and
+implements most of the methods you need from the :ref:`PyTorchTrial
+<pytorch-trial>` API for you. Since in this approach the
+``LightningModule`` is not paired with the standard trainer that comes
+with Pytorch Lightning some methods and hooks are not supported. Read
+about those here:
```

Comment:
```suggestion
In this approach, the ``LightningModule`` is not paired with the ``Trainer`` so that there are some methods and hooks that are not supported. Read about those here:
```

Reply:
I updated this but I didn't switch the whole paragraph just the last sentence. is that what you meant?